app/models/container_image.rb

```diff
@@ -6,6 +6,7 @@ class ContainerImage < ApplicationRecord
 
 
   DOCKER_IMAGE_PREFIX = "docker://"
+  DOCKER_PULLABLE_PREFIX = "docker-pullable://"
```

Comment:
I am almost sure that it is. The patch that is introducing it to kuberentes was merged but has since been reverted, we will have to see what happens there but there was an agreement on this part

[1] https://github.com/kubernetes/kubernetes/pull/33014/files#diff-0d64fd1b097fff84be9e16633e6acf20R46